entity-framework/core/miscellaneous/nativeaot-and-precompiled-queries.md

```diff
@@ -0,0 +1,110 @@
+---
+title: NativeAOT Support and Precompiled Queries (Experimental) - EF Core
+description: Publishing NativeAOT Entity Framework Core applications and using precompiled queries
+author: roji
+ms.date: 11/10/2024
+uid: core/miscellaneous/nativeaot-and-precompiled-queries
```

Comment:
Shouldn't this be under `core/performance`?

Reply:
Good point! Moving.